secrets.yaml.sample

```diff
@@ -4,3 +4,8 @@ alertmanager:
 logging:
   esS3AccessKey: example
   esS3SecretKey: example
+# If you are using X-Pack of elastic you must change below values
```

Comment:
Please change to 'If you are using Elastic X-Pack you must change the below values'